Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM debian:bookworm
```

Comment:
Would it be better to base this on a somewhat extended scratch or distroless?

Reply:
I was thinking that some users might want the possibility to work from inside the container, that is what I went with using Debian as base.

Reply:
As this is a tool for bioinformaticians and not a service, I think people might appreciate to have debian as a base in case they want to shell into it or use it in workflows ☺️